package.json

```diff
@@ -64,6 +64,7 @@
     "cozy-client": "1.0.0-beta.4",
     "cozy-scripts": "0.5.5",
     "cozy-ui": "9.0.1",
+    "cozy-vcard": "^0.2.18",
```

Comment:
*cozy-vcard* is a vcard parser, while the `src/importation/vcard` module is a vcard importer, i.e. it takes the output of some parser, converts it as needed and saves it somewhere.

Since snapshot tests & most fixtures were removed in #168, remaining ones are only unit tests for our own `src/importation/*` modules.

Does it help?